From fa76f4309a9ea8d3d2296d3775679e67e0b17512 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Fri, 23 Oct 2020 07:33:14 -0500
Subject: [PATCH 71/71] arm64: dts: beacon-imx8m m/n baseboards: Fix OTG VBUS
 pinmux

In order to properly pinmux a GPIO pin, both pintrl-names and pinctrl-0
need to be defined.  These boards were missing the pinctrl-names entry
meaning the corresponding pinctrl-0 was never invoked leading to erratic
or inconsistent behavior on the OTG VBUS

Add the missing entry to ensure proper muxing of the VBUS regulator
GPIO.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi | 1 +
 arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
index 7abd4b97dd09..14509da58cd7 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
@@ -108,6 +108,7 @@
 
 	reg_usb_otg_vbus: usb_otg_vbus {
 		compatible = "regulator-fixed";
+		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_reg_usb_otg>;
 		regulator-name = "usb_otg_vbus";
 		regulator-min-microvolt = <5000000>;
diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi
index 4aea29e38524..af0852ace6e6 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mn-baseboard.dtsi
@@ -90,6 +90,7 @@
 
 	reg_usb_otg_vbus: usb_otg_vbus {
 		compatible = "regulator-fixed";
+		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_reg_usb_otg>;
 		regulator-name = "usb_otg_vbus";
 		regulator-min-microvolt = <5000000>;
-- 
2.17.1

